nss-cohort-36/keahua-arboretum-net-surfers
# Jl feed edits

# Description

redid feed animal section - orig was creating new animal every time and not saving.  corrected to where it feeds an existing animal as designed

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jl-feed-edits`
4. create minimum of 2 biomes and animals in them - select which you'd like to feed and choose the meal the animal is having

# Checklist:
